.github/workflows/deploy-apis-to-production.yml

```diff
@@ -367,6 +367,16 @@ jobs:
           cat k8s/kafka/connectors/measurements-connect.yaml | sed "s/{{MEASUREMENTS_CONNECT_IMAGE}}/$MEASUREMENTS_CONNECT_IMAGE/g" | sed "s/{{KAFKA_CLUSTER}}/$PROD_KAFKA_CLUSTER/g" | kubectl apply -n message-broker -f-
           kubectl apply -f k8s/kafka/connectors/purple-air-connector.yaml -n message-broker
 
+      - name: Set up GCP credentials
+        run : |
+          echo "${{ secrets.prod-env-kafka-bigquery-connectors }}" > google_application_credentials.json
+          echo "${{ secrets.prod-key-kafka-bigquery-connectors }}" > bigquery_credentials.json
+
```

Comment:
Could we use the secrets from GCP secret manager?

Reply:
Done as requested @123MwanjeMike